README.md

```diff
@@ -63,19 +63,25 @@ gcloud beta container clusters create bank-of-anthos \
 --tags=bank-of-anthos --labels csm=
 ```
 
-4. **Deploy the demo JWT public key** to the cluster as a Secret. This key is used for user account creation and authentication.
+4. **Get credentials for the created cluster**
+
+```
+gcloud container clusters get-credentials bank-of-anthos --project=${PROJECT} --zone=${ZONE}
```

Comment:
Thanks for this PR, Yuki! One small change request - to align with the commands above, could you please replace `PROJECT` with `PROJECT_ID` ? thank you!

Reply:
Oops! Sorry fixed that.